docker/start-local-environment.sh

```diff
@@ -3,9 +3,9 @@ set -x -e
 
 
 #these two lines of code, makes the docker work with intellij (Windows modification)
-#docker-compose down --rmi=local --remove-orphans
-#docker-compose rm
-#
+docker-compose down --rmi=local --remove-orphans
+docker-compose rm
+
```

Comment:
Was this meant to be committed?

Reply:
Not sure about this one @SRAlexander @ellenwootten are you aware of this change?

Reply:
this wasn't me! Scott?